addons/account/wizard/account_payment_register.py

```diff
@@ -589,6 +610,16 @@ def _onchange_amount(self):
     def _onchange_payment_date(self):
         self.wizard_change_mode = 'payment_date_change'
 
+    @api.onchange('payment_method_line_id')
+    def _onchange_payment_method_line_id(self):
```

Comment:
Adding this onchange and not in the compute of journal_id because journal_id is a precompute field and i don't think we want to put the payment_method_line as a precompute field for the depends